README.CentOS7.md

```diff
@@ -103,6 +103,31 @@ Customize and Verify your settings:
  * DocumentRoot should point to `/var/www/loris/htdocs`
  * The `smarty/templates_c/` directory must be writable by Apache (e.g. by running: `sudo chgrp -R httpd /var/www/loris/smarty/templates_c` and `sudo chmod 775 /var/www/loris/smarty/templates_c`).
 
+Modify the Apache configuration file for our production environment.
+```
+vi /etc/httpd/conf/httpd.conf
+```
+
+**a.** Find & modify the `ServerName` to:
+```
+ServerName your.Loris.url.here:443
```

Comment:
This document doesn't have any instruction on how to setup SSL that I see, so this port is wrong.

Reply:
@driusan I changed it to 80 in latest commit.

Reply:
there's no reason to include a port number at all in the server name.

Reply:
I think it's fine to prompt `(typically port 443 for SSL)`